stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>Add links to Overwatch and OWL. A few pictures throughout would be nice too if possible (e.g. of different maps, of a tournament,   any other concepts that are introduced in the text later).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I agree strongly with this. Adding in some screenshots would really help to better understand what this story is about.</p>

Reply:
<p>Good idea, I sprinkled some screenshots and such around the story.</p>

Reply:
The new screenshots and images look great.